arm/Microsoft.OperationalInsights/workspaces/.parameters/parameters.json

```diff
@@ -17,8 +17,8 @@
         "storageInsightsConfigs": {
             "value": [
                 {
-                    "name": "zwmqrvlchzw5i",
-                    "storageAccountId": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Storage/storageAccounts/zwmqrvlchzw5i",
+                    "name": "adpsxxazsaweux003",
```

Comment:
If that is always the storage name we could maybe retrieve it in the template itself by something like 

`name: last(split(storageAccountId, '/'))`

so that we get rid of a parameter. What's your take on that?

UPDATE: I see we already have a var for that `storageAccountName `in the storageInsightsConfigs module so we could leverage that for the `name`

Reply:
Agree, updated!